src/AppContestMultiSeat.test.tsx

```diff
@@ -79,12 +79,10 @@ it('Single Seat Contest', async () => {
   expect(container.firstChild).toMatchSnapshot()
 
   // Go to Review Screen
-  while (!queryByText('Review →')) {
+  while (!queryByText('All Your Votes')) {
```

Comment:
does it still say "All your votes" now? Should it say all your choices? Or is this appearing elsewhere?

Reply:
Here is a screenshot where you can see all this language:

![image](https://user-images.githubusercontent.com/28444/65527831-2fd6f780-dea8-11e9-9f40-2043ca795c72.png)

Reply:
Decided to revert to using "votes" and be sensitive to see if any voters on the review screen believe that they are casting more votes than they are supposed to.